kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -292,27 +292,34 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
       throw new IllegalStateException("Methods initialize and isSupported must be called ahead")
     }
     debug(s"Getting application[${toLabel(tag)}]'s info from Kubernetes cluster")
+    val startTime = System.currentTimeMillis()
```

Comment:
this is used to prevent the elapsedTime too long caused by metadataManager or other slowness